cli/azd/internal/appdetect/java.go

```diff
@@ -145,9 +145,12 @@ func detectDependencies(mavenProject *mavenProject, project *Project) (*Project,
 		}
 	}
 	applicationProperties := make(map[string]string)
+	var springBootVersion string
 	if isSpringBoot {
+		springBootVersion = detectSpringBootVersion(currentRoot, mavenProject)
 		applicationProperties = readProperties(project.Path)
 	}
+	_ = springBootVersion
```

Comment:
What is this used for?

Reply:
not used for now, but I will use it in the next fix PR, in which it will inject a configuration based on the spring boot version.

Reply:
How about adding it when it's used?

Reply:
Agree that.